src/RenderConsole.js

```diff
@@ -38,10 +38,10 @@ module.exports = class RenderConsole {
         output += `
   Vulnerabilities:`
       }
-
-      vulnerabilitiesResults.details.items.forEach(vulnItem => {
+      const that = this
```

Comment:
I don't like this style :)
can you please revert the change. Why did you need to do it to begin with?